src/pip/_internal/operations/install/wheel.py

```diff
@@ -324,6 +327,110 @@ def make(self, specification, options=None):
         return super(PipScriptMaker, self).make(specification, options)
 
 
+def _get_file_owners(
+        lib_dir,      # type: str
+        top_level,    # type: str
+        ignore_name,  # type: str
+):
+    # type: (...) -> Dict[str, List[str]]
```

Comment:
Nitpick, but I believe we can fit the arguments in one line without making flake8 unhappy.
```suggestion
def _get_file_owners(lib_dir, top_level, ignore_name):
    # type: (str, str, str) -> Dict[str, List[str]]
```